src/ChooseAGameName/StoryPiece.java

```diff
@@ -20,6 +20,7 @@
 	private String story;
 	private boolean isDanger;
 	private boolean hasBeenThere;
+	JPanel panel;
```

Comment:
Instance variables should be private, unless there's a reason for them to be something else (e.g. public, protected).

Reply:
I didn't think I was using this so I deleted it....my game works just find without it.